Fix document price search (#36)
ATM-Consulting/dolibarr_module_discountrules: ChangeLog.md

## Not Released

- NEW : Last price search in document configuration *12/12/2021* - 2.14.0  

>   The search in document feature is to get the last practiced price for a customer  

Reviewer: ```suggestion
  The search in document feature finds the last price applied to a customer  
```

>   A conf to keep possibility to search best price instead of last price

Reviewer: ```suggestion
  A conf enables the module to search the best price instead of the last price
```
